README.md

```diff
@@ -59,7 +59,7 @@ from tensorbay.dataset import Data, Dataset
 dataset = Dataset("DatasetName")
 
 # TensorBay uses "segment" to separate different parts in a dataset.
-segment = dataset.create_segment()
+segment = dataset.create_segment("SegmentName")
```

Comment:
there are may examples in our docs, read segment with empty str `""`, should be updated